topics/RDMA.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "RDMA support"
+linkTitle: "RDMA support"
+weight: 1
+description: Valkey Over RDMA support
+aliases: [
+    /topics/RDMA,
+    /docs/manual/RDMA,
+    /docs/manual/RDMA.md
+]
```

Comment:
This is just left-overs from redis docs, because they renamed the pages many times and these were used for creating redirects. We don't need that.

```suggestion
```